blocks/tree-view/tree-view.js

```diff
@@ -160,7 +160,7 @@ const init = async (el) => {
     topList.role = 'menu';
 
     const topListItems = topList.querySelectorAll(':scope > li');
-
+    console.log(' topListItems ', topListItems);
```

Comment:
HI @karri-sek ,

Could you please remove the console.log ? I believe this was added just for debugging ?